ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -115,11 +116,12 @@ def decelerate_waypoints(self, waypoints, closest_idx):
     def pose_cb(self, msg):
         self.pose = msg
 
-    def position(waypoint):
-        position = waypoint.pose.pose.position
-        return [position.x, position.y]
-
     def waypoints_cb(self, waypoints):
+	self.base_lane = waypoints
+        def position(waypoint):
```

Comment:
This doesn't need to be a local function, it can be re-used 'get_closest_waypoint_index' too